examples/vite/public/example.jpg

```diff
@@ -0,0 +1 @@
+/Users/edmunddao/_SPACE/EyePop.ai/Untitled/eyepop-sdk-node/examples/example.jpg
```

Comment:
This seems to be an erroneous commit.

Reply:
Also a sym link, resolved via [1b14f62](https://github.com/eyepop-ai/eyepop-sdk-node/pull/31/commits/1b14f6251cf82dc803abbcdb14a2710bb675b917)